### PR TITLE
Hotfix Docker Release Pipeline

### PR DESCRIPTION
**Pull request by agustin-mrtz**
Fix the file not found error that is blocking the deployment. Removing the `$(pwd)`when defining the path for the Dockerfiles since they're all in the root directory

### CODE REVIEW

File: Jenkinsfile.release [hunk 1]
```diff
@@ -83,16 +83,16 @@ dockerizedBuildPipeline(
 )
 
 void updateIQServerVersionAndChecksum(String version, String checksum) {
-  def dockerFile = readFile(file: "${pwd()}/Dockerfile")
+  def dockerFile = readFile(file: "Dockerfile")
```
Comment (tneer) [inline Jenkinsfile.release]: Minor nit:  In Groovy we generally prefer single quoted strings when no string interpolation is done or needed.

**Review by tneer — APPROVED** [review]
LGTM

File: Jenkinsfile.release (hunk 1 above)
Comment (agustin-mrtz) [reply Jenkinsfile.release]: Addressed: https://github.com/sonatype/docker-nexus-iq-server/pull/125/commits/a9c1b3f5e5d616dabba80da21893298aa2e30c31

**Review by kakumara — APPROVED** [review]
+1, LGTM

**Pull request merged**